<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="VideoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>426</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>426</width>
      <height>341</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>426</width>
      <height>341</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="VideoLayout">
     <item>
      <widget class="GLViewWidget" name="Video" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>426</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1080</width>
         <height>720</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>1081</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="ControlsButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VideoButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ButtonWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QPushButton" name="TestButton">
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>44</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="TestSlider">
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>270</width>
         <height>44</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.opengl</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
